perdiem.py

```diff
@@ -150,23 +149,31 @@ def select_rate(records, city):
 def request_gsa_destination(state, city, zipcode, fiscal_year):
     """
     Make a request to the GSA API.
-
     GSA API doc: https://www.gsa.gov/technology/government-it-initiatives/digital-strategy/per-diem-apis/per-diem-api
     """
-    gsa_url = "https://inventory.data.gov/api/action/datastore_search"
-    payload = {
-        'resource_id': '8ea44bc4-22ba-4386-b84c-1494ab28964b',
-        'limit': 20,
-        'filters': '{{"FiscalYear":"{}","Zip":"{}","State":"{}"}}'.format(fiscal_year, zipcode, state),
-    }
-    r = requests.get(gsa_url, params=payload)
+    gsa_url = f'https://api.gsa.gov/travel/perdiem/v2/rates/zip/{zipcode}/year/{fiscal_year}'     #old url"https://inventory.data.gov/api/action/datastore_search"
+    headers = {'x-api-key':'zoXm1gdyKjNAr6SIIJnd42u9ZvVbNSZvuPTmW1zV'}
```

Comment:
**question**: is this api key ok to be in the public or should it be a secret?

Reply:
I think it's ok, but I can't say for sure. I think it's like our api where it's tracking usage.

Reply:
I would suggest if there is any question that it then be removed.